end_to_end/gemma/7b/1_test_gemma.sh

```diff
@@ -7,25 +7,33 @@
 # 1. Pull the checkpoint from a GCS bucket and uploads the new MaxText compatible checkpoint to destination GCS bucket.
 # 2. Convert the scanned checkpoint from step 1 into unscanned checkpoint format and run more efficient decoding.
 
-# Example Usage: bash end_to_end/gemma/7b/1_test_gemma.sh
+# Example Usage: export BASE_OUTPUT_DIRECTORY=/path/to/GCS/bucket; bash end_to_end/gemma/7b/1_test_gemma.sh
```

Comment:
Nit:
In MaxText land, `BASE_OUTPUT_DIRECTORY` is usually just a gcs bucket, we attach the run_name to it and create a unique path for the run. 
I think we should be more clear in the example usage here, saying that BASE_OUTPUT_DIRECTORY is the full path of user's unique run. 

I think we should also consider renaming that variable to `BASE_OUTPUT_PATH` to avoid confusion.

Reply:
Thanks Mohit!

I added a comment in the scripts to clarify and also renamed to `BASE_OUTPUT_PATH`.